base/permuteddimsarray.jl

```diff
@@ -16,6 +16,29 @@ immutable PermutedDimsArray{T,N,perm,iperm,AA<:AbstractArray} <: AbstractArray{T
     end
 end
 
+"""
+    PermutedDimsArray(A, perm) -> B
+
+Given an AbstractArray `A`, create a view `B` such that the
+dimensions appear to be permuted. Similar to `permutedims`, except
+that no copying occurs (`B` shares storage with `A`).
+
+See also: [`permutedims`](@ref).
+
+# Example
+
+```jldoctest
+julia> A = rand(3,5,4);
+
+julia> B = PermutedDimsArray(A, (3,1,2));
+
+julia> size(B)
+(4,3,5)
```

Comment:
this needs to be:
```julia
(4, 3, 5)
```
to pass the doctest after https://github.com/JuliaLang/julia/pull/20288